lib/Item/ItemAccessibilitySupport.js

```diff
@@ -18,11 +18,39 @@ module.exports = {
 	/**
 	* @private
 	*/
+	handlers: {
+		onRequestMarqueeStart  : 'marqueeStartHandler',
+		onSpotlightBlur : 'spotlightBlurdHandler'
+	},
+	
+	/**
+	* @private
+	*/
 	updateAccessibilityAttributes: kind.inherit(function (sup) {
 		return function (was, is, prop) {
 			var enabled = !this.accessibilityDisabled;
 			sup.apply(this, arguments);
 			this.setAttribute('aria-labelledby', !this.accessibilityLabel && enabled ? this.getId() : null);
 		};
-	})
+	}),
+
+	/**
+	* @private
+	*/
+	marqueeStartHandler: function (oSender, oEvent) {
+		var enabled = !this.accessibilityDisabled;
+		if (this._marquee_distance !== 0 && !this.$.marqueeText && enabled) {
```

Comment:
Relying on private members from another mixin will make this brittle. I'm not sure what the 'less-brittle' solution is here. It feels like we need to someone better integrate accessibility with Marquee rather than relying on the Item mixin to handle it. Thoughts?